sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -247,10 +257,10 @@ public void cancelOperation() throws UnsupportedOperationException {
      * @return A {@link Mono} that returns {@link PollResponse}. This will call poll operation once.
      */
     public Mono<PollResponse<T>> poll() {
-        return this.pollOperation.apply(this.pollResponse)
+        return pollOperation.apply(pollResponse.get())
```

Comment:
earlier in my review I received to comment to use  this for class variable. Also this which class uses this all other places. So for consistency, shouldn't we use `this.` here as well.

Reply:
I had reviews where for terseness and it does't conflict, to prefer not to use this.

Reply:
I would rather remove `this.`. For me, it only adds cognitive burden.

Reply:
ok good for me.

Reply:
I reverted this back.